internal/cmd/controller/grutil/resourcekey.go

```diff
@@ -124,64 +129,58 @@ func toType(resource fleet.GitRepoResource) (string, string) {
 	return t, resource.Namespace + "/" + resource.Name
 }
 
-func addState(bundle fleet.Bundle, resources map[fleet.ResourceKey][]fleet.ResourcePerClusterState) (bool, []error) {
+func addState(bd fleet.BundleDeployment, resources map[fleet.ResourceKey][]fleet.ResourcePerClusterState) (bool, []error) {
 	var (
 		incomplete bool
 		errors     []error
 	)
 
-	if len(bundle.Status.Summary.NonReadyResources) >= 10 {
+	if len(bd.Status.NonReadyStatus) >= 10 {
```

Comment:
Why do we not take `bd.Status.ModifiedStatus` into account here, as we used to?

Reply:
I mush have overlooked it. It's back now